base-template.sh

```diff
@@ -684,7 +696,7 @@ update_shared_hooks_if_appropriate() {
 
                 # shellcheck disable=SC2181
                 if [ $? -ne 0 ]; then
-                    echo "! Update failed, git pull output:" >&2
+                    echo "WARNING: Shared hook update failed, git pull output:" >&2
```

Comment:
Can you please revert formatting changes like this one and the one above to keep output formatting consistent?